src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/CopyLocaleToolbarAction.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+import React from 'react';
+import {action, observable} from 'mobx';
+import jexl from 'jexl';
+import Checkbox from '../../../components/Checkbox';
+import Dialog from '../../../components/Dialog';
+import ResourceRequester from '../../../services/ResourceRequester';
+import {translate} from '../../../utils/Translator';
+import AbstractFormToolbarAction from './AbstractFormToolbarAction';
+import copyLocaleActionStyles from './copyLocaleAction.scss';
+
+export default class CopyLocaleToolbarAction extends AbstractFormToolbarAction {
+    @observable showCopyLocaleDialog = false;
+    @observable selectedLocales: Array<string> = [];
+    @observable copying: boolean = false;
+
+    getNode() {
+        const {
+            resourceFormStore: {
+                data: {
+                    availableLocales,
+                },
+                id,
+                locale: currentLocale,
+            },
+            locales,
+        } = this;
+
+        if (!id) {
+            return null;
+        }
+
+        if (!locales || !currentLocale) {
+            throw new Error('The CopyLocaleToolbarAction for pages only works with locales!');
+        }
+
+        return (
+            <Dialog
+                cancelText={translate('sulu_admin.cancel')}
+                confirmLoading={this.copying}
+                confirmText={translate('sulu_admin.ok')}
+                key="sulu_admin.copy_locale"
+                onCancel={this.handleClose}
+                onConfirm={this.handleConfirm}
+                open={this.showCopyLocaleDialog}
+                title={translate('sulu_admin.copy_locale')}
+            >
+                <div className={copyLocaleActionStyles.dialog}>
+                    <p>{translate('sulu_admin.choose_target_locale')}:</p>
+                    {locales.map((locale) => currentLocale.get() === locale
```

Comment:
Nothing todo with this PR but `currentLocale.get()` looks for me like a pitfall for other developers should we not return a string instead of the observable as computed value?